src/jqLite.js

```diff
@@ -215,7 +229,10 @@ function jqLiteBuildFragment(html, context) {
     tmp = fragment.appendChild(context.createElement('div'));
     tag = (TAG_NAME_REGEXP.exec(html) || ['', ''])[1].toLowerCase();
     wrap = wrapMap[tag] || wrapMap._default;
```

Comment:
You could get rid of `_default`, e.g. like this:

```javascript
wrap = wrapMap[tag]
finalHTML = ...
tmp.innerHTML = wrap ? wrap[1] + finalHtml + wrap[2]  : finalHTML;
i = wrap ? wrap[0] : 0;
while (i--) { //...
```

That's a no-op for the security fix, but is shorter - and lets you pipe through the data without any string concatenation (e.g. letting through `TrustedHTML` objects that DOMPurify might return).

Reply:
Sound sensible. That said, this would technically be a new feature and we're at the point where we mostly care about security fixes so I wouldn't want to spend too much time on tweaking this code.

Reply:
OK, apparently we have a few days to finish the PR; in that case, I can have a look early next weekend.